assistant/seeder/generate_prompt_sqlite.py

```diff
@@ -0,0 +1,55 @@
+import os
+import sqlite3
+import pandas as pd
+
+from db import DB_NAME, DB_PATH
+from typing import Optional
+
+FILENAME = "prompt"
+
+
+def main(db_name: Optional[str] = DB_NAME):
+    csv_file = f"./sources/{FILENAME}.csv"
+    sqlite_db = DB_PATH.format(db_name=db_name)
+
+    if not os.path.exists(csv_file):
+        print(f"404 - File not found: {csv_file}")
+        return None
+
+    try:
+        prompt_detail = pd.read_csv(csv_file)
+        prompt_group_df = pd.DataFrame(
+            {
+                "id": range(1, len(prompt_detail["stable"].unique()) + 1),
+                "stable": prompt_detail["stable"].unique(),
+            }
+        )
+        prompt_detail["prompt_id"] = prompt_detail["stable"].map(
+            prompt_group_df.set_index("stable")["id"]
+        )
+        prompt_detail = prompt_detail.drop(columns=["stable"])
+        print(prompt_detail)
+    except Exception as e:
+        print(f"Error reading CSV: {e}")
```

Comment:
Please use `logger` and not `print`. Not all print statements make it out of a container.